src/coresyftools/coresyftools/tests/test_gpt_tool.py

```diff
@@ -76,10 +76,13 @@ def __call__(self, args):
 
     def test_graph(self):
         manifest = self.manifest.copy()
-        manifest['operation'] = {'graph': True}
+        graph_file_name = 'my_tool.gpt_graph.xml'
+        manifest['operation'] = {
+            'graph': graph_file_name
+        }
```

Comment:
There steps could be move into "setUp()" method.

Reply:
This is specific to this test, if I moved it to `setUp` all tests would have a _graph_, and we only want it in this test.